### PR TITLE
task(SDK-3204) - Adds debug key-value to meta object in header of payload

### PR DESCRIPTION
**Pull request by Anush-Shand**
This addition is part of the integration debugger feature

### CODE REVIEW

File: clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.java [hunk 1]
```diff
@@ -519,6 +519,10 @@ String insertHeader(Context context, JSONArray arr,@Nullable final String caller
             }
             // Add frs (First Request in Session) and update first request flag
             header.put("frs", coreMetaData.isFirstRequestInSession());
+
+            if(config.getDebugLevel() == 3)
```
Comment (piyush-kukadiya) [inline clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.java:523]: @Anush-Shand Just add comment, mentioning use of this header.

Comment (darshanclevertap) [reply clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.java:524]: @Anush-Shand is there a possibility of config.getDebugLevel being NULL?

**Review by piyush-kukadiya — APPROVED** [review]
👍

File: clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.java (hunk 1 above)
Comment (Anush-Shand) [reply clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.java:524]: @darshanclevertap Nope, I don't think so

**Review by darshanclevertap — APPROVED** [review]
Approved but don't merge to develop as of now.

**Review by darshanclevertap — APPROVED** [review]

**Pull request merged**